src/react/workflow/ConfigurationTab.tsx

```diff
@@ -64,65 +58,101 @@ const ConfigurationHeader = styled.div`
 
 function ConfigurationTab({
   wfSelected,
-  replications,
   bucketList,
   locations,
 }: Props) {
   const dispatch = useDispatch();
   const history = useHistory();
   const queryClient = useQueryClient();
   const { workflowId } = wfSelected;
-  const isDeleteModalOpen = useSelector(
-    (state: AppState) => state.uiWorkflows.showWorkflowDeleteModal,
-  );
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const mgnt = useManagementClient();
+  const state = useSelector((state: AppState) => state);
+  const { instanceId } = getClients(state);
+  const accountId = getAccountId(state);
+
+  const replicationsQuery = useQuery({
+    ...workflowListQuery(
+      notFalsyTypeGuard(mgnt),
+      accountId,
```

Comment:
I prefer not taking this approach and refactor the way we retrieved these info so that their typing is actually the expected one at the level of this component, however as this may take a bit of time that I don't have for now, I think it is fine to merge as it is now